tools/data_manipulation/read_netcdf/read_netcdf/netcdf_read.xml

```diff
@@ -0,0 +1,259 @@
+<tool id="netcdf_read" name="Netcdf Reader" version="0.2.0">
+    <description>extracts variable values with custom conditions on dimensions</description>
+    <requirements>
+        <requirement type="package" version="2.1.0">matplotlib</requirement>
+        <requirement type="package" version="1.3.1">netCDF4</requirement>
+        <requirement type="package" version="1.0.0">scipy</requirement>
+        <requirement type="package" version="1.1.0">datamash</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_dir && 
+
+    #if $condi_source_coord.coord_source=="coord_from_file"
+        i=0 &&
+        re='^[-+]?[0-9]+\.?[0-9]*$' && 
+        while read line; do 
```

Comment:
would this not be better in a python script? It would be way faster as well ...

Reply:
sorry, Alan designed it like this. Is it possible to look at it as a enhancement in the future?

Reply:
Or maybe I can give a rapid try, but this is cheetah, not really directly Python isn't it ?

Reply:
I really meant a python script, that you ship with the tool instead of this gross, but powerful bash hacks.

Reply:
Ok, if you think this will be better, notably in term of performance, I will ask a Python guy to look at it! Thank you !!!

Reply:
The Python guy is @jusana ;) At a first glance, it seems that the cheetah code seems to be efficient...